doc/user/css/magidoc-overrides.css

```diff
@@ -0,0 +1,11 @@
+/*
+ * This file contains custom CSS overrides for the GraphQL documentation available at
+ * https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/
+ */
+
+/*
+ * Hide the deprecated queries from the left hand navigation panel.
+ */
+nav ul li .deprecated {
+  display: none;
+}
```

Comment:
How are they displayed without this? I'm just wondering since sometimes we use the deprecated on experimental queries (like we did for planConnection in the past), and I'm not sure if we want to hide their existence completely.

Reply:
![image](https://github.com/user-attachments/assets/4566e4d9-4f3e-4ba7-8939-b17e1f4d0e34)

It removes them from the left hand navigation but you can still access them from the search and if you type in the URL.